### DISCUSSION TITLE
Request for updated ViTSTR model link

### DISCUSSION THREAD

**Discussion by machoangha** [PaddlePaddle/PaddleOCR#13946] [Q&A] [answered]
Hi,

I was trying to access the SVITR model, but it seems like the link is no longer available. Could you please provide an updated link or let me know where I can find it?

Thanks in advance for your help!

Best regards,

**Comment by GreatV** [comment]
which link?

**Reply by machoangha** [reply]
Hi, Im sorry the model is ViTSTR:
https://github.com/PaddlePaddle/PaddleOCR/blob/main/doc/doc_en/algorithm_rec_vitstr_en.md
The link to download the weights is down.

**Reply by GreatV** [reply, marked as answer]
https://paddleocr.bj.bcebos.com/rec_vitstr_none_ce_train.tar
